openslides/motions/static/js/motions/site.js

```diff
@@ -515,10 +528,35 @@ angular.module('OpenSlidesApp.motions.site', ['OpenSlidesApp.motions'])
                 });
         };
 
+        // Export the given motions as a csv file
+        $scope.csv_export = function (motions) {
```

Comment:
I think we should not use a function argument `motion` here but `$scope.motionsFiltered`. Another possibility is to move this logic to a service outside the controller so we can reuse it anywhere else if needed.